keps/sig-storage/2924-csi-migration-cephfs/README.md

```diff
@@ -35,6 +41,32 @@ used as described in its parent KEP. For all other contents, please refer to the
   on kubelet and treat Ceph Cephfs CSI migration as already complete. And directly redirect traffic to CSI 
   driver for all cephfs related operations.
 
+## Design Details
+
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+No additional tests are needed, rather the issue is orchestrating CSI driver
+deployment for prow jobs. This means that it is not possible to run any test for
+cephfs in k/k repository.
+
+##### Unit tests
+
+N/A
+
+##### Integration tests
+
+N/A
+
+##### e2e tests
+
+Support for tests after cephfs migration will be performed by the subjected
+cephfs csi driver.
```

Comment:
Can you point to the repo and folder where the relevant e2e tests will be added?